### DISCUSSION TITLE
Including same .md file in "nav" structure on different places results bad behaviour

### DISCUSSION THREAD

**Discussion by TGD-HW** [mkdocs/mkdocs#3596] [Q&A] [answered]
Hi guys,
I'm asking for little help here as I'm new to MkDocs and markdown.
I'm developing a simple user manual web here:

https://tgd-hw.github.io/tgmanGH/

and I use single source "electrical.md" file within different categories in "nav":

![obrazek](https://github.com/mkdocs/mkdocs/assets/163130991/8b61a7e8-2c15-47d5-a7b1-98a087975df3)
 with intention to have only one source file which I can edit and changes apply everywhere.
But MkDocs behavior is not what I would like as it is confused which category to open when clicking on any of the "Elektrická instalace" page. 

![obrazek](https://github.com/mkdocs/mkdocs/assets/163130991/7365970e-4f32-48cf-ac4a-c96b42ea6e6f)

Can you please advise some solution?
Thanks a lot.

**Comment by pawamoy** [comment, marked as answer]
You could simulate different pages, but with the same contents, with the help of [pymdownx.snippets](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/): create three distinct markdown pages in each TGZ... folder, and put this in each:

```md
--8<-- "Manual/source/CZ/md/electrical.md"
```

Optionally move this file somewhere else.

Then update your `nav` to use these distinct pages rather than the original one.

**Reply by TGD-HW** [reply]
Thanks a lot. It works very well.
I also added a base path to snippets:
![obrazek](https://github.com/mkdocs/mkdocs/assets/163130991/ae97b799-dd5f-4654-83ed-59d94a340a2a)
Regards
